capstone-backend/src/main/java/com/google/sps/servlets/MeetingServlet.java

```diff
@@ -122,6 +126,10 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       organizer.setEmail(jsonObject.get("organizerEmail").getAsString());
     }
     event.setOrganizer(organizer);
+    if (keySet.contains("recurrence") && jsonObject.get("recurrence").getAsString().length() > 0) {
+      String[] recurrence = new String[] {jsonObject.get("recurrence").getAsString()};
```

Comment:
Interesting, so events can have "multiple recurrences"?

Reply:
I believe they can, since this is how the API guides handle recurrences, but in our case we only allow them to have one.